src/views/BaseView.vue

```diff
@@ -208,5 +209,6 @@
 }
 pre {
   white-space: pre-line;
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Comment:
@cesarvh If you add `class="body-2"` to the <pre> element, you won't need to hard-code the font-family here.